packages/config/src/doc/IProfArgAttrs.ts

```diff
@@ -15,7 +15,7 @@ import { IProfLoc } from "./IProfLoc";
 export type IProfDataType = "string" | "number" | "boolean" | "array" | "object";
 
 /** The value for the argument */
-export type IProfArgValue = string | number | boolean | string[] | object;
+export type IProfArgValue = string | number | boolean | string[] | object | undefined;
```

Comment:
I'm not sure we want to add `undefined` here, since it's not a valid type for profile arguments.

I'd suggest making the change here instead: https://github.com/zowe/imperative/blob/ca981d476b7a97fa6129de523c63e1a58000c139/packages/config/src/doc/IProfInfoUpdatePropOpts.ts#L53-L56

Reply:
Changed